.setup/INSTALL_SUBMITTY_HELPER_SITE.sh

```diff
@@ -77,6 +77,12 @@ for i in "${array[@]}"; do
     find ${SUBMITTY_INSTALL_DIR}/site/public -type f -name \*.${i} -exec chmod o+r {} \;
 done
 
+#Setup Cron Jobs
+crontab -r -u submitty_daemon
```

Comment:
Let's not be totally destructive on cron in case we plan to utilize it elsewhere.

Reply:
I was removing the current jobs list just because if not, every time a user runs install, a copy of same job would be appended to the list. 
Is there are better place to put this command? Maybe we can standardize an area for cron jobs in one of the install scripts?

Reply:
```
crontab -u submitty_daemon -l > /tmp/cron_jobs
grep "python3 ${Submitty_INSTALL_DIR}/sbin/send_email.py" /tmp/cron_jobs || echo "* * * * * python3 ${SUBMITTY_INSTALL_DIR}/sbin/send_email.py" >> /tmp/cron_jobs
crontab -u submitty_daemon -r
crontab -u submitty_daemon /tmp/cron_jobs
rm -f /tmp/cron_jobs
```

A longer discussion would be do we really need to progromatically manipulate this crontab, or would we be better off just creating a static one that we put in say `.setup/crontabs/` and that we always just install that file. I think I'd really prefer this option as it also makes explicitly clear in the codebase how we might add more cron jobs for a particular user.